src/app/models/types/onward-journey.ts

```diff
@@ -0,0 +1,126 @@
+import { OptimoBlock } from './optimo';
+
+export type OptimoRecommendation = {
+  headlines: {
+    seoHeadline?: string;
+    promoHeadline: {
+      blocks: OptimoBlock[];
+    };
+  };
+  locators: {
+    optimoUrn: string;
+    canonicalUrl: string;
+  };
+  summary: {
+    blocks: OptimoBlock[];
+  };
+  images: {
+    defaultPromoImage: {
+      blocks: OptimoBlock[];
+    };
+  };
+  timestamp: number;
+  suitableForSyndication?: boolean;
+  language?: string;
+  passport?: object;
+  id?: string;
+  type?: string;
+};
+
+export type CpsRecommendation = {
+  headlines: {
+    shortHeadline?: string;
+    headline: string;
+  };
+  locators: {
+    assetUri: string;
+    cpsUrn?: string;
+  };
+  summary: string;
+  timestamp: number;
+  indexImage: IndexImage;
+  cpsType?: string;
+  id?: string;
+  type?: string;
+};
+
+export type Recommendation = OptimoRecommendation | CpsRecommendation;
+
+export type TopStoriesOJ = {
```

Comment:
Could we write this in longhand? 
```suggestion
export type TopStoriesOnwardJourney = {
```